### DISCUSSION TITLE
False Positive Malware in pyqt/6.0.3_1 ?

### DISCUSSION THREAD

**Discussion by hpwxf** [Homebrew/discussions#1186] [Everyday usage]
Hello Everybody,

Today I upgrade my brew (macOS Big Sur v11.2.3, x86-64) and BitDefender detects the malware `Gen:Variant.Application.IOS.ZergHelperCA.1` inside `/usr/local/Cellar/pyqt/6.0.3_1/lib/python3.9/site-packages/PyQt6/QtGui.abi3.so` (this file comes from `https://homebrew.bintray.com/bottles/pyqt-builder-1.9.1.big_sur.bottle.tar.gz`).

I got this file from quarantine and submit it to TotalVirus (online version):
![Snapshot](https://user-images.githubusercontent.com/26146722/113472838-fb150a80-9465-11eb-80f1-dd3dbe755f3a.png)
([full report](https://www.virustotal.com/gui/file/a5cfce8ab6c89e483eeac7ab952a33e84ba49ee7e2f59e054a0a388cd31848ed/detection))

Thus, the same malware is detected as "malicious" by 9 security vendors (and not the most famous).

I tried to rebuild from source the `pyqt` package using `brew reinstall -s pyqt` and this time I got the same malware inside a temporary build file: `/private/tmp/tmp92siujzf/QtGui/sipQtGuiQFontDatabase.o` (it stopped the compilation process).

Since it appears in a temporary file (built from source), I assume that is a false positive. Do you agree?

**Comment by SMillerDev** [comment]
I'm no security expert so I can't really say anything about this but if your local install yielded the same result it's unlikely to be a compromise on the Homebrew side. I'd ask this question to the PyQT developers instead (or an actual security expert)

**Reply by hpwxf** [reply]
Ok; I subscribed to the PyQt mailing list and I posted the same message.

If I get an answer, I will post it here.
